src/test/java/org/fundacionjala/coding/cynthia/EanValidationTest.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.cynthia;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+/**
+ * Created by USUARIO on 05/06/2017.
+ */
+public class EanValidationTest {
+    private EanValidation eanValidation = new EanValidation();
```

Comment:
this is not a good practice. you can not instantiate object directly in the attributes of class.
try this.
```java
private EanValidation eanValidation;

    /**
     * Initial setup.
     */
    @Before
    public void setUp() {
        eanValidation = new EanValidation();
    }
```